nrfcloud/calculateCostsPerAccount.ts

```diff
@@ -0,0 +1,27 @@
+import { calculateCosts } from './calculateCosts.js'
+import { SSMClient } from '@aws-sdk/client-ssm'
+import { getAllnRFCloudAccounts } from './allAccounts.js'
+import { getCostSummaryFromAPI } from './getCostSummaryFromAPI.js'
+
+export type AccountObject = { [key: string]: number }
+
+export const calculateCostsPerAccount = async ({
+	ssm,
+	stackName,
+	date,
+}: {
+	ssm: SSMClient
+	stackName: string
+	date: number
+}): Promise<AccountObject> => {
```

Comment:
Change the lambda function to not use the global `getAllnRFCloudAccounts()` but the function passed as the argument.